### PR TITLE
Change flags to opt in to being enabled for stages.

### PR DESCRIPTION
**Pull request by microbit-matt-hillsdon**
It's no longer appropriate to enable all flags by default for REVIEW
stage.

Closes https://github.com/microbit-foundation/python-editor-next/issues/827

### CODE REVIEW

File: src/common/sanity.ts [hunk 1]
```diff
@@ -58,7 +58,7 @@ export const fetchContent = async <T>(
 };
 
 const fetchContentInternal = async (query: string): Promise<any> => {
-  const response = await fetch(queryUrl(query));
+  const response = await fetch(queryUrl(query), { credentials: "include" });
```
Comment (microbit-matt-hillsdon) [inline src/common/sanity.ts:61]: Oops, unrelated change.

```suggestion
  const response = await fetch(queryUrl(query));
```

File: src/documentation/ideas/content.ts [hunk 2]
```diff
@@ -14,7 +14,7 @@ const ideasQuery = (languageId: string): string => {
     throw new Error("Invalid language id.");
   }
   return `
-    *[_type == "pythonIdea" && language == "${languageId}" && !(_id in path("drafts.**"))]{
+    *[_type == "pythonIdea" && language == "${languageId}" ]{
```
Comment (microbit-matt-hillsdon) [inline src/documentation/ideas/content.ts:17]:
```suggestion
    *[_type == "pythonIdea" && language == "${languageId}" && !(_id in path("drafts.**"))]{
```

**Review by microbit-robert — CHANGES_REQUESTED** [review]
Please see comment above.

**Comment by microbit-matt-hillsdon** [comment]
> Using `flag=none` does not seem to disable flags enabled by stage. Using `flag=*` seems to disable all flags rather than enabling them.

Added tests and fix in 465041ab90a7229b099e38d8134ae4c1394d3d00. The logic is also easier to follow now.

**Review by microbit-robert — APPROVED** [review]
Thanks. LGTM.
